.teamcity/src/main/kotlin/promotion/BasePromotionBuildType.kt

```diff
@@ -43,6 +43,9 @@ abstract class BasePromotionBuildType(vcsRootId: String, cleanCheckout: Boolean
             param("env.GRADLE_CACHE_REMOTE_PASSWORD", "%gradle.cache.remote.password%")
             param("env.GRADLE_CACHE_REMOTE_URL", "%gradle.cache.remote.url%")
             param("env.GRADLE_CACHE_REMOTE_USERNAME", "%gradle.cache.remote.username%")
+            // https://www.jetbrains.com/help/teamcity/shared-resources.html#Viewing+Shared+Resources+Usage
+            // we only allow 1 promotion job running at the same time to avoid website xml conflicts
+            param("teamcity.locks.writeLock.WebsiteReleasesXmlWriteLock", "WebsiteReleasesXmlWriteLock")
```

Comment:
I'd call the resource `WebsiteReleasesXmlResource`. Then there can be read on write locks on the same resource, depending on how it is referenced, right? WDYT?